packages/core/src/Stores/notification-store.js

```diff
@@ -1477,6 +1480,10 @@ export default class NotificationStore extends BaseStore {
         this.p2p_order_props = p2p_order_props;
     }
 
+    setP2PRedirectTo(p2p_redirect_to) {
```

Comment:
@ameerul-deriv Not sure if it's a good idea to make `NotificationStore` aware of routing in `@deriv/p2p`, Couldn't we handle this by a `useEffect` in the `@deriv/p2p` itself? 🤔

Reply:
this would definitely be a better option after I have removed p2p from cashier, as I can directly access notification store and maybe have a flag to check if the user had clicked that button... but for now there isn't a way for me to direct the user to my profile page in p2p when user is in cashier...

Reply:
I had done something similar before but for redirecting to order details page from outside of p2p. But it only redirects them to the orders page